setup.py

```diff
@@ -25,7 +25,9 @@ def read(*paths):
     'python-dateutil>=2.4.0',
     'rfc3986>=0.3.0',
     'jsonschema>=2.5.1',
-    'future>=0.15.2'
+    'future>=0.15.2',
+    'tabulator>=0.3',
+    'unicodecsv>=0.14',
```

Comment:
Do we need this? If anything, we should only need it for Py2, but AFAIK the code for this is about this simple: https://github.com/okfn/goodtables/blob/master/goodtables/compat.py#L30